.idea/vcs.xml

```diff
@@ -2,6 +2,5 @@
 <project version="4">
   <component name="VcsDirectoryMappings">
     <mapping directory="$PROJECT_DIR$" vcs="Git" />
-    <mapping directory="$PROJECT_DIR$/src/locustio" vcs="Git" />
```

Comment:
Is this on purpose?

Reply:
Nope. The file shouldn't be in the repository in the first place so I'll remove it.